samples/SimpleCheckout/app/build.gradle

```diff
@@ -24,6 +25,21 @@ private String getVersionName() {
     return file("./version").text.trim()
 }
 
+apollo {
+    service("storefront") {
+        packageName.set("com.shopify.checkout_sdk_sample")
+
+        mapScalar("URL", "java.net.URL")
+        mapScalarToKotlinDouble("Decimal")
+
+        introspection {
+            endpointUrl.set("https://$storefrontDomain/api/2024-10/graphql.json")
```

Comment:
It would be nice to take the API version from the env too, but fall back to a static value

```
def storefrontApiVersion = properties.getProperty("STOREFRONT_API_VERSION") ?? "2024-10"

endpointUrl.set("https://$storefrontDomain/api/$storefrontApiVersion/graphql.json")
```